test/e2e/tests/transaction/send-eth.spec.js

```diff
@@ -18,11 +17,10 @@ describe('Send ETH', function () {
       await withFixtures(
         {
           fixtures: new FixtureBuilder().build(),
-          ganacheOptions: defaultGanacheOptions,
           title: this.test.fullTitle(),
```

Comment:
this spec is using anvil.
All tests that use defaultGanacheOptions, once migrated to anvil, won't be a need to pass defaultXYZOptions, since those will be already default in the anvil config, so we can remove this.